provisioning/group_vars/castlemind.yml

```diff
@@ -0,0 +1,51 @@
+---
+app_root: /home/vagrant
+
+nodeapp_name: castlemind
+nodeapp_user: vagrant
+nodeapp_index: '{{ app_root }}/{{ nodeapp_name }}/bin/www'
+nodeapp_node_version: 0.10.33
+nodeapp_num_workers: 2
+nodeapp_env:
+  NODE_ENV: production
+  PORT: "`printf '32%02i' $NODEAPP_INDEX`"
+
+nginx_http_params:
+  gzip_comp_level: 6
+  gzip_vary: 'on'
+  gzip_min_length: 1000
+  gzip_proxied: any
+  gzip_types: 'text/plain text/html text/css application/json application/x-javascript text/xml application/xml application/xml+rss text/javascript'
+  gzip_buffers: '16 8k'
+
+  proxy_cache_path: '/var/cache/nginx levels=1:2 keys_zone=one:8m max_size=3000m inactive=600m'
+  proxy_temp_path: '/var/tmp'
+
+nginx_upstreams:
+  - name: app_proxy
+    servers:
+      - 127.0.0.1:3201
+      - 127.0.0.1:3202
+
+nginx_sites:
+ - server:
+    file_name: castlemind
+    listen: 80
+    root: '{{ app_root }}/{{nodeapp_name}}'
+
+    error_page: 404 /errors/404.html
+    error_page: 500 501 502 503 504 /errors/5xx.html
```

Comment:
These errors would be shown if the node process dies but `nginx` is still handling incoming requests. We don't actually have a `public/errors/5xx.html` file checked in, but it would be an easy add.